pyproject.toml

```
@@ -62,6 +62,7 @@ docs = [
     "riscemu==2.2.7",
     "micropip>=0.7",
 ]
+llvmlite = ["llvmlite==0.46.0"]
```

Also let's just call this llvm, in case we want to swap out for other bindings in the future while preserving compatibility
```suggestion
llvm = ["llvmlite==0.46.0"]
```